Resources/Prototypes/_Goobstation/Entities/Clothing/Shoes/specific.yml

```
@@ -0,0 +1,28 @@
+    sprintModifier: 1.1
+  - type: Skates
+    stunSeconds: 0
+    speedDamage: 0
+    minimumSpeed: 999
+  - type: FootstepModifier
+    footstepSoundCollection:
+      collection: FootstepSkates
+      params:
+        variation: 0.08
+  - type: DamageOnHighSpeedImpact
+    minimumSpeed: 999
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)